.github/workflows/ci.yml

```
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  pull_request:
+
+env:
+  node_version: 20.x
+
+jobs:
+  style:
+    name: Lint and Format
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.node_version }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Check formatting
+        run: npm run format:check
```

I'll give more comments to help you set up linting.
```suggestion
      - name: Check formatting
        run: npm run format:check
        
      - name: Lint code
        run: npm run lint
```